package.json

```diff
@@ -15,11 +15,12 @@
     "typescript": "^3.1.6"
   },
   "scripts": {
-    "lint": "tslint -p ./tsconfig.json",
+    "lint": "tslint --fix -p ./tsconfig.json",
```

Comment:
i'd rather not have `--fix` turned on for CI, can we make this a different command? or specify --fix manually?

Reply:
ok, fixed the `--fix` 😄